src/lib/AutoControlledComponent.js

```diff
@@ -54,7 +54,7 @@ export default class AutoControlledComponent extends Component {
       } else if (prop === 'checked') {
         res[prop] = false
       } else if (prop === 'value') {
-        res[prop] = ''
+        res[prop] = this.props.multiple ? [] : '' // eslint-disable-line react/prop-types
```

Comment:
A select element value must be an array if `multiple` is true.  Dropdowns render a hidden select.  We need to use an empty array as the default value when multiple, instead of an empty string.